### PR TITLE
1616 cloud public cloud

### LINKED ISSUE #1616: /cloud/public-cloud/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /cloud/public-cloud to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/public-cloud)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-public-cloud.png)

## Issue / Card

Fixes #1616

### CODE REVIEW

**Comment by yaili** [comment]
Design review:
- [ ] Remove border bottom from hero
- [ ] Same comment regarding CTA in contextual footer (can it move above)
- [ ] And same comment regarding button neutral in contextual footer

**Comment by pmahnke** [comment]
@yaili updated

File: static/sass/_v1_pattern_card.scss [hunk 1]
```diff
@@ -38,4 +38,22 @@
...
+      a:link,
+      a:visited {
+        color: $color-brand;
+      }
+
+      a:hover,
+      a:active {
+        color: $color-brand;
+        text-decoration: underline;
+      }
```
Comment (anthonydillon) [inline static/sass/_v1_pattern_card.scss]: Can you make sure to file an issue upstream for this and comment this with a XXX and a link to the issue.

File: templates/cloud/_nav_tertiary-v1.html [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-    {% endif %}
-    {% if 'contact-us' in level_4 %}
-    <li><a {% if level_4 == 'contact-us' %} class="is-active"{% endif %} href="/{{level_1}}/{{level_2}}/{{level_3}}/{{level_4}}">Contact us</a></li>
-    {% endif %}
+<ul class="p-inline-list nav-tertiary__menu">
+  {% if level_2 == 'openstack' and not level_4 %}
+  <li class="p-inline-list__item"><a{% if level_3 == 'autopilot'%} class="is-active"{% endif %} href="/cloud/openstack/autopilot">Autopilot</a></li>
+  {% endif %}
+  {% if level_3 == 'managed-cloud' and level_4 %}
+  <li class="p-inline-list__item" class="p-inline-list__item"><a href="/{{level_1}}/{{level_2}}/{{level_3}}">Managed cloud</a></li>
```
Comment (anthonydillon) [inline templates/cloud/_nav_tertiary-v1.html]: Double class

File: templates/cloud/_nav_tertiary-v1.html [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+  {% endif %}
+  {% if 'thank-you' in level_3 %}
+  <li class="p-inline-list__item"><a{% if level_3 == 'thank-you' %} class="is-active"{% endif %}href="/{{level_1}}/{{level_2}}/{{level_3}}">Thank you</a></li>
+  {% endif %}
+  {% if 'contact-us' in level_3 %}
+  <li class="p-inline-list__item"><a {% if level_3 == 'contact-us' %} class="is-active"{% endif %} href="/{{level_1}}/{{level_2}}/{{level_3}}">Contact us</a></li>
+  {% endif %}
+  {% if 'thank-you' in level_4 %}
+  <li class="p-inline-list__item" class="p-inline-list__item"><a{% if level_4 == 'thank-you' %} class="is-active"{% endif %}href="/{{level_1}}/{{level_2}}/{{level_3}}/{{level_4}}">Thank you</a></li>
+  {% endif %}
```
Comment (anthonydillon) [inline templates/cloud/_nav_tertiary-v1.html]: Double class

File: templates/cloud/public-cloud.html [hunk 4]
```diff
@@ -1,94 +1,99 @@
...
-            <p>Answers to your most frequently asked questions about using Ubuntu as a cloud guest.</p>
-            <p class="content-cat content-cat-white-paper">White paper</p>
-        </a>
-        </div>
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-12">
+      <h2>Why use Ubuntu Server in the cloud?</h2>
+      </div>
+      </div>
```
Comment (anthonydillon) [inline templates/cloud/public-cloud.html]: Under indented

File: templates/cloud/public-cloud.html [hunk 5]
```diff
@@ -1,94 +1,99 @@
...
-            <h2>Ready to get started?</h2>
-            <p>Ubuntu Server {{lts_release_full}} is a free download that includes five years worth of  security and maintenance updates, guaranteed. Using an Ubuntu partner cloud ensures access to the latest certified Ubuntu cloud images. To run Ubuntu as guest on participating public clouds, the simplest way is to use the image maintained on their server.</p>
-        </div><!-- /.eight-col -->
+<section class="p-strip--light is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h2>Ready to get started?</h2>
+      <p>Ubuntu Server {{lts_release_full}} is a free download that includes five years worth of  security and maintenance updates, guaranteed. Using an Ubuntu partner cloud ensures access to the latest certified Ubuntu cloud images. To run Ubuntu as guest on participating public clouds, the simplest way is to use the image maintained on their server.</p>
+    </div>
+    </div>
```
Comment (anthonydillon) [inline templates/cloud/public-cloud.html]: Over indented

File: templates/cloud/shared/_guest-list-v1.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
+<div class="row">
+  <div class="u-equal-height">
+    <div class="col-4 p-card">
+      <heading class="p-card__header u-align--center u-vertically-center" style="min-height: 8rem;">
+        <img style="max-width: 150px;" src="{{ ASSET_SERVER_URL }}bd47d87c-google-cloud.svg" alt="Google Cloud Platform logo" />
```
Comment (anthonydillon) [inline templates/cloud/shared/_guest-list-v1.html]: Does this need to be an inline style?

File: templates/cloud/shared/_guest-list-v1.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
+<div class="row">
+  <div class="u-equal-height">
+    <div class="col-4 p-card">
+      <heading class="p-card__header u-align--center u-vertically-center" style="min-height: 8rem;">
```
Comment (anthonydillon) [inline templates/cloud/shared/_guest-list-v1.html]: Heading?

File: static/sass/_v1_pattern_card.scss (hunk 1 above)
Comment (pmahnke) [inline static/sass/_v1_pattern_card.scss]: done [bug](https://github.com/vanilla-framework/vanilla-brochure-theme/issues/82)

File: templates/cloud/_nav_tertiary-v1.html (hunk 2 above)
Comment (pmahnke) [inline templates/cloud/_nav_tertiary-v1.html]: done

File: templates/cloud/_nav_tertiary-v1.html (hunk 3 above)
Comment (pmahnke) [inline templates/cloud/_nav_tertiary-v1.html]: done

File: templates/cloud/public-cloud.html (hunk 4 above)
Comment (pmahnke) [inline templates/cloud/public-cloud.html]: done

File: templates/cloud/public-cloud.html (hunk 5 above)
Comment (pmahnke) [inline templates/cloud/public-cloud.html]: done

File: templates/cloud/shared/_guest-list-v1.html (hunk 7 above)
Comment (pmahnke) [inline templates/cloud/shared/_guest-list-v1.html]: done

File: templates/cloud/shared/_guest-list-v1.html (hunk 6 above)
Comment (pmahnke) [inline templates/cloud/shared/_guest-list-v1.html]: I think so... all these logos are different...

**Comment by yaili** [comment]
@pmahnke I refreshed the demo a few times but still see everything as before

**Comment by deadlight** [comment]
@yaili the demo is updated

**Comment by deadlight** [comment]
@pmahnke There's a styling bug on this resource card:
whitetext

**Comment by pmahnke** [comment]
@deadlight fixed by pulling in master

File: run [hunk 8]
```diff
@@ -230,27 +230,27 @@ case $run_command in
             node_run "${project}-watch" "yarn run watch" "--detach"  # Run watch in the background
         fi
 
-        
+
 
         # Run the serve container, publishing the port, and detaching if required
-        django_run "${project}-serve" "" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach} ${run_serve_docker_opts} ${*}"
```
Comment (nottrobin) [inline run]: Where did this change come from? I think @WillMoggridge might need this old line to remain for his demo work...

File: templates/cloud/_nav_secondary-v1.html [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
-    <li><a{% if level_1 == 'cloud' and not level_2 %} class="is-active"{% endif %} href="/cloud">Overview</a></li>
-    <li><a{% if level_2 == 'openstack' %} class="is-active"{% endif %} href="/cloud/openstack">OpenStack</a></li>
-    <li><a{% if level_2 == 'managed-cloud' %} class="is-active"{% endif %} href="/cloud/managed-cloud">Managed cloud</a></li>
-    <li><a{% if level_2 == 'public-cloud' %} class="is-active"{% endif %} href="/cloud/public-cloud">Public cloud</a></li>
-    <li><a{% if level_2 == 'juju' %} class="is-active"{% endif %} href="/cloud/juju">Juju</a></li>
-    <li><a{% if level_2 == 'storage' %} class="is-active"{% endif %} href="/cloud/storage">Storage</a></li>
-    <li><a{% if level_2 == 'partners' %} class="is-active"{% endif %} href="/cloud/partners">Partners</a></li>
-    <li><a{% if level_2 == 'training' %} class="is-active"{% endif %} href="/cloud/training">Training</a></li>
-    <li><a class="last-item{% if level_2 == 'plans-and-pricing' %} is-active{% endif %}" href="/cloud/plans-and-pricing">Plans and pricing</a></li>
+<ul class="{% if list_class %}{{ list_class }}{% else %}nav-secondary__menu p-inline-list{% endif %}">
```
Comment (nottrobin) [inline templates/cloud/_nav_secondary-v1.html]: These files no longer exist - please remove them

File: templates/cloud/_nav_tertiary-v1.html [hunk 10]
```diff
@@ -1,23 +1,25 @@
...
-    {% if 'contact-us' in level_3 %}
-    <li><a {% if level_3 == 'contact-us' %} class="is-active"{% endif %} href="/{{level_1}}/{{level_2}}/{{level_3}}">Contact us</a></li>
-    {% endif %}
-    {% if 'thank-you' in level_4 %}
-    <li><a{% if level_4 == 'thank-you' %} class="is-active"{% endif %}href="/{{level_1}}/{{level_2}}/{{level_3}}/{{level_4}}">Thank you</a></li>
-    {% endif %}
-    {% if 'contact-us' in level_4 %}
-    <li><a {% if level_4 == 'contact-us' %} class="is-active"{% endif %} href="/{{level_1}}/{{level_2}}/{{level_3}}/{{level_4}}">Contact us</a></li>
-    {% endif %}
+{% if level_3 %}
```
Comment (nottrobin) [inline templates/cloud/_nav_tertiary-v1.html]: Same - please remove this file

File: templates/cloud/contact-us.html [hunk 11]
```diff
@@ -1,61 +1,42 @@
-{% extends "cloud/base_cloud.html" %}
+{% extends "cloud/base_cloud-v1.html" %}
 {% block title %}Contact us | Ubuntu Cloud{% endblock %}
 
 {% block second_level_nav_items %}
```
Comment (nottrobin) [inline templates/cloud/contact-us.html]: This block is a thing of the past

**Comment by nottrobin** [comment]
This is now looking far more sane, I'm approving.

**Pull request merged**